### PR TITLE
enable first_inference_latency (first_inference+time_to_inference)

### PR DESCRIPTION
**Pull request by AlbertAnglesIntel**
### Details:
 - Update run_timetest.py , to include the time_to_first_inference_result = time_to_inference + first_inference, in aggregated statistics
 
with model_cache (FIL)-time_to_first_inference_result expressed in ms:

C:\>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -c -f c:\OpenVINO\compiler_cid_latency.conf -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
[ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c -f C:\OpenVINO\compiler_cid_latency.conf -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpnstx_19n
[ INFO ] ========== Completed. Exit code: 0
[ DEBUG ] Statistics after run of executable #0: {'full_run': 202516, 'time_to_inference': 182542, 'load_plugin': 50339, 'create_exenetwork': 121357, 'load_network_cache': 121354, 'first_inference': 13416, 'fill_inputs': 2421}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 202516, 'stdev': 0}, 'time_to_inference': {'avg': 182542, 'stdev': 0}, 'load_plugin': {'avg': 50339, 'stdev': 0}, 'create_exenetwork': {'avg': 121357, 'stdev': 0}, 'load_network_cache': {'avg': 121354, 'stdev': 0}, 'first_inference': {'avg': 13416, 'stdev': 0}, 'fill_inputs': {'avg': 2421, 'stdev': 0}, 'time_to_first_inference_result (ms)': {'avg': 195.958}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 121357, 'stdev': 0},
 'fill_inputs': {'avg': 2421, 'stdev': 0},
 'first_inference': {'avg': 13416, 'stdev': 0},
 'full_run': {'avg': 202516, 'stdev': 0},
 'load_network_cache': {'avg': 121354, 'stdev': 0},
 'load_plugin': {'avg': 50339, 'stdev': 0},
 '**time_to_first_inference_result** (ms)': {'avg': 195.958},
 'time_to_inference': {'avg': 182542, 'stdev': 0}}

C:\>

without (FEIL)-> expressed in seconds

C:\>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -f c:\OpenVINO\compiler_cid_latency.conf -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
[ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX  -f C:\OpenVINO\compiler_cid_latency.conf -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpt_7li_0q
[ INFO ] ========== Completed. Exit code: 0
[ DEBUG ] Statistics after run of executable #0: {'full_run': 1768380.0, 'time_to_inference': 1747320.0, 'load_plugin': 52984, 'create_exenetwork': 1680630.0, 'read_network': 17096, 'load_network': 1663530.0, 'first_inference': 13223, 'fill_inputs': 2408}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 1768380.0, 'stdev': 0}, 'time_to_inference': {'avg': 1747320.0, 'stdev': 0}, 'load_plugin': {'avg': 52984, 'stdev': 0}, 'create_exenetwork': {'avg': 1680630.0, 'stdev': 0}, 'read_network': {'avg': 17096, 'stdev': 0}, 'load_network': {'avg': 1663530.0, 'stdev': 0}, 'first_inference': {'avg': 13223, 'stdev': 0}, 'fill_inputs': {'avg': 2408, 'stdev': 0}, 'time_to_first_inference_result (s)': {'avg': 1.760543}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 1680630.0, 'stdev': 0},
 'fill_inputs': {'avg': 2408, 'stdev': 0},
 'first_inference': {'avg': 13223, 'stdev': 0},
 'full_run': {'avg': 1768380.0, 'stdev': 0},
 'load_network': {'avg': 1663530.0, 'stdev': 0},
 'load_plugin': {'avg': 52984, 'stdev': 0},
 'read_network': {'avg': 17096, 'stdev': 0},
 '**time_to_first_inference_result** (s)': {'avg': 1.760543},
 'time_to_inference': {'avg': 1747320.0, 'stdev': 0}}

### Tickets:
 - https://jira.devtools.intel.com/browse/EISW-64751

### CODE REVIEW

**Comment by AlbertAnglesIntel** [comment]
the solution proposed here was a work around/temporary solution. 
As suggested by @dkrupnov best is to enable it in 
https://github.com/openvinotoolkit/openvino/blob/releases/2022/3/tests/time_tests/src/timetests/timetest_infer.cpp#L29 

WIP

File: tests/time_tests/src/timetests/timetest_infer.cpp [hunk 1]
```diff
@@ -27,51 +27,54 @@ int runPipeline(const std::string &model, const std::string &device, const bool
         size_t batchSize = 0;
 
         // first_inference_latency = time_to_inference + first_inference
+        SCOPED_TIMER(first_inference_latency);
```
Comment (evgenytalanin-intel) [inline tests/time_tests/src/timetests/timetest_infer.cpp:30]: Should be moved inside the next visibility scope

File: tests/time_tests/scripts/run_timetest.py [hunk 2]
```diff
@@ -49,11 +49,17 @@ def parse_stats(stats: list, res: dict):
...
 
 
-def aggregate_stats(stats: dict):
+def aggregate_stats(stats: dict, args:dict):
     """Aggregate provided statistics"""
-    return {step_name: {"avg": statistics.mean(duration_list),
+    agg_stats = {step_name: {"avg": statistics.mean(duration_list),
                         "stdev": statistics.stdev(duration_list) if len(duration_list) > 1 else 0}
             for step_name, duration_list in stats.items()}
+    if args["model_cache"]: # FIL-First Inference Latency
```
Comment (evgenytalanin-intel) [inline tests/time_tests/scripts/run_timetest.py:57]: Do not need this as FIL is prepared in .cpp file below

File: tests/time_tests/scripts/run_timetest.py [hunk 3]
```diff
@@ -49,11 +49,17 @@ def parse_stats(stats: list, res: dict):
...
+        agg_stats['time_to_first_inference_result']={'avg': (agg_stats['time_to_inference']['avg']+agg_stats['first_inference']['avg']) / 10 **3}
+    else: #FEIL-First Ever Inference Latency
+        agg_stats['time_to_first_inference_result']={'avg': (agg_stats['time_to_inference']['avg']+agg_stats['first_inference']['avg']) / 10 **6}
```
Comment (evgenytalanin-intel) [inline tests/time_tests/scripts/run_timetest.py:60]: Just for the record: absolute strange to divide by 1e3 or 1e6 depending on some key. This makes results from different runs hard to compare. Also use 1e3 and 1e6 as legit numbers, eliminating power operation every time

**Review by evgenytalanin-intel — CHANGES_REQUESTED** [review]

File: tests/time_tests/scripts/run_timetest.py (hunk 2 above)
Comment (AlbertAnglesIntel) [reply tests/time_tests/scripts/run_timetest.py]: agree, removed

File: tests/time_tests/scripts/run_timetest.py (hunk 3 above)
Comment (AlbertAnglesIntel) [reply tests/time_tests/scripts/run_timetest.py]: I will leave as default time unit (microsends) to be consistent with the rest.

File: tests/time_tests/src/timetests/timetest_infer.cpp (hunk 1 above)
Comment (AlbertAnglesIntel) [reply tests/time_tests/src/timetests/timetest_infer.cpp:30]: done , moved inside { } 

however, when executed I see:

[setupvars.bat] OpenVINO environment initialized
[ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c -f C:\OpenVINO\compiler_cid_latency.conf -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpgm5ojze2
[ INFO ] ERROR: unknown command line flag 'f'
[ INFO ] ========== Completed. Exit code: 1
[ ERROR ] Run of executable 'timetest_infer.exe' failed with return code '1'. Error: ERROR: unknown command line flag 'f'

Comment (AlbertAnglesIntel) [reply tests/time_tests/src/timetests/timetest_infer.cpp:30]: my bad:  -f is not a recognized flag

C:\Users\Local_Admin>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -c -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
cmd_common: ['C:\\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\\tools\\timetest\\timetest\\scripts\\timetest_infer.exe', '-m', 'C:\\models\\mobilenet-v2\\onnx\\FP16\\mobilenet-v2.xml', '-d', 'VPUX', '-c', '', ''][ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c   -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpjzl3v24j
[ INFO ] ========== Completed. Exit code: 0
retcode: 0
msg:
raw_data: [[{'full_run': [2554470.0, {'first_inference_latency': [2362160.0, {'time_to_inference': [2296500.0, {'load_plugin': [684983]}, {'create_exenetwork': [1572610.0, {'load_network_cache': [1572610.0]}]}]}, {'first_inference': [65660, {'fill_inputs': [4168]}]}]}]}], {'measurement_unit': 'microsecs'}]
[ DEBUG ] Statistics after run of executable #0: {'full_run': 2554470.0, 'first_inference_latency': 2362160.0, 'time_to_inference': 2296500.0, 'load_plugin': 684983, 'create_exenetwork': 1572610.0, 'load_network_cache': 1572610.0, 'first_inference': 65660, 'fill_inputs': 4168}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 2554470.0, 'stdev': 0}, 'first_inference_latency': {'avg': 2362160.0, 'stdev': 0}, 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}, 'load_plugin': {'avg': 684983, 'stdev': 0}, 'create_exenetwork': {'avg': 1572610.0, 'stdev': 0}, 'load_network_cache': {'avg': 1572610.0, 'stdev': 0}, 'first_inference': {'avg': 65660, 'stdev': 0}, 'fill_inputs': {'avg': 4168, 'stdev': 0}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 1572610.0, 'stdev': 0},
 'fill_inputs': {'avg': 4168, 'stdev': 0},
 'first_inference': {'avg': 65660, 'stdev': 0},
 '**first_inference_latency**': {'avg': 2362160.0, 'stdev': 0},
 'full_run': {'avg': 2554470.0, 'stdev': 0},
 'load_network_cache': {'avg': 1572610.0, 'stdev': 0},
 'load_plugin': {'avg': 684983, 'stdev': 0},
 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}}

**Comment by AlbertAnglesIntel** [comment]
ready using SCOPED_REGION:

C:\Users\Local_Admin>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -c -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
cmd_common: ['C:\\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\\tools\\timetest\\timetest\\scripts\\timetest_infer.exe', '-m', 'C:\\models\\mobilenet-v2\\onnx\\FP16\\mobilenet-v2.xml', '-d', 'VPUX', '-c', '', ''][ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c   -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpjzl3v24j
[ INFO ] ========== Completed. Exit code: 0
retcode: 0
msg:
raw_data: [[{'full_run': [2554470.0, {'first_inference_latency': [2362160.0, {'time_to_inference': [2296500.0, {'load_plugin': [684983]}, {'create_exenetwork': [1572610.0, {'load_network_cache': [1572610.0]}]}]}, {'first_inference': [65660, {'fill_inputs': [4168]}]}]}]}], {'measurement_unit': 'microsecs'}]
[ DEBUG ] Statistics after run of executable #0: {'full_run': 2554470.0, 'first_inference_latency': 2362160.0, 'time_to_inference': 2296500.0, 'load_plugin': 684983, 'create_exenetwork': 1572610.0, 'load_network_cache': 1572610.0, 'first_inference': 65660, 'fill_inputs': 4168}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 2554470.0, 'stdev': 0}, 'first_inference_latency': {'avg': 2362160.0, 'stdev': 0}, 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}, 'load_plugin': {'avg': 684983, 'stdev': 0}, 'create_exenetwork': {'avg': 1572610.0, 'stdev': 0}, 'load_network_cache': {'avg': 1572610.0, 'stdev': 0}, 'first_inference': {'avg': 65660, 'stdev': 0}, 'fill_inputs': {'avg': 4168, 'stdev': 0}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 1572610.0, 'stdev': 0},
 'fill_inputs': {'avg': 4168, 'stdev': 0},
 'first_inference': {'avg': 65660, 'stdev': 0},
 '**first_inference_latency**': {'avg': 2362160.0, 'stdev': 0},
 'full_run': {'avg': 2554470.0, 'stdev': 0},
 'load_network_cache': {'avg': 1572610.0, 'stdev': 0},
 'load_plugin': {'avg': 684983, 'stdev': 0},
 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}}

**Comment by AlbertAnglesIntel** [comment]
> ready using SCOPED_REGION:
> 
> C:\Users\Local_Admin>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -c -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml" Python 3.9.12 [setupvars.bat] OpenVINO environment initialized cmd_common: ['C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe', '-m', 'C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml', '-d', 'VPUX', '-c', '', ''][ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpjzl3v24j [ INFO ] ========== Completed. Exit code: 0 retcode: 0 msg: raw_data: [[{'full_run': [2554470.0, {'first_inference_latency': [2362160.0, {'time_to_inference': [2296500.0, {'load_plugin': [684983]}, {'create_exenetwork': [1572610.0, {'load_network_cache': [1572610.0]}]}]}, {'first_inference': [65660, {'fill_inputs': [4168]}]}]}]}], {'measurement_unit': 'microsecs'}] [ DEBUG ] Statistics after run of executable #0: {'full_run': 2554470.0, 'first_inference_latency': 2362160.0, 'time_to_inference': 2296500.0, 'load_plugin': 684983, 'create_exenetwork': 1572610.0, 'load_network_cache': 1572610.0, 'first_inference': 65660, 'fill_inputs': 4168} [ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 2554470.0, 'stdev': 0}, 'first_inference_latency': {'avg': 2362160.0, 'stdev': 0}, 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}, 'load_plugin': {'avg': 684983, 'stdev': 0}, 'create_exenetwork': {'avg': 1572610.0, 'stdev': 0}, 'load_network_cache': {'avg': 1572610.0, 'stdev': 0}, 'first_inference': {'avg': 65660, 'stdev': 0}, 'fill_inputs': {'avg': 4168, 'stdev': 0}} [ INFO ] Aggregated statistics: {'create_exenetwork': {'avg': 1572610.0, 'stdev': 0}, 'fill_inputs': {'avg': 4168, 'stdev': 0}, 'first_inference': {'avg': 65660, 'stdev': 0}, '**first_inference_latency**': {'avg': 2362160.0, 'stdev': 0}, 'full_run': {'avg': 2554470.0, 'stdev': 0}, 'load_network_cache': {'avg': 1572610.0, 'stdev': 0}, 'load_plugin': {'avg': 684983, 'stdev': 0}, 'time_to_inference': {'avg': 2296500.0, 'stdev': 0}}
> 
> as we can see:
> 
> first_inference_latency=first_inference+time_to_inference

**Comment by AlbertAnglesIntel** [comment]
in another execution : 

C:\Users\Local_Admin>cmd /V /C "C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208\setupvars.bat & cd C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\\timetest\scripts & python run_timetest.py timetest_infer.exe -d VPUX -niter 1 -c -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
cmd_common: ['C:\\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\\tools\\timetest\\timetest\\scripts\\timetest_infer.exe', '-m', 'C:\\models\\mobilenet-v2\\onnx\\FP16\\mobilenet-v2.xml', '-d', 'VPUX', '-c', '', '']
[ INFO ] ========== cmd: C:\w_ov_dyn_win_t_221208_2351_a6d0a3c71b_ci_tag_vpux_nightly_20221208_TOOLS\tools\timetest\timetest\scripts\timetest_infer.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -c   -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmpqd1fdrhq
[ INFO ] ========== Completed. Exit code: 0
retcode: 0
msg:
raw_data: [[{'full_run': [148485, {'first_inference_latency': [139812, {'time_to_inference': [126099, {'load_plugin': [40233]}, {'create_exenetwork': [69027, {'load_network_cache': [69023]}]}]}, {'first_inference': [13707, {'fill_inputs': [2594]}]}]}]}], {'measurement_unit': 'microsecs'}]
[ DEBUG ] Statistics after run of executable #0: {'full_run': 148485, 'first_inference_latency': 139812, 'time_to_inference': 126099, 'load_plugin': 40233, 'create_exenetwork': 69027, 'load_network_cache': 69023, 'first_inference': 13707, 'fill_inputs': 2594}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 148485, 'stdev': 0}, 'first_inference_latency': {'avg': 139812, 'stdev': 0}, 'time_to_inference': {'avg': 126099, 'stdev': 0}, 'load_plugin': {'avg': 40233, 'stdev': 0}, 'create_exenetwork': {'avg': 69027, 'stdev': 0}, 'load_network_cache': {'avg': 69023, 'stdev': 0}, 'first_inference': {'avg': 13707, 'stdev': 0}, 'fill_inputs': {'avg': 2594, 'stdev': 0}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 69027, 'stdev': 0},
 'fill_inputs': {'avg': 2594, 'stdev': 0},
 'first_inference': {'avg': 13707, 'stdev': 0},
 '**first_inference_latency**': {'avg': 139812, 'stdev': 0},
 'full_run': {'avg': 148485, 'stdev': 0},
 'load_network_cache': {'avg': 69023, 'stdev': 0},
 'load_plugin': {'avg': 40233, 'stdev': 0},
 'time_to_inference': {'avg': 126099, 'stdev': 0}}

first_inference_latency~ (close to )= first_inference+time_to_inference

**Review by vurusovs — APPROVED** [review]

**Comment by AlbertAnglesIntel** [comment]
using timetest_infer_api_2.exe:

c:>cmd /V /C "C:\w_ov_dyn_win_t_230109_2200_2022_3_0_ci_tag_vpux_nightly_20230109\setupvars.bat & cd C:\w_ov_dyn_win_t_230109_2200_2022_3_0_ci_tag_vpux_nightly_20230109_TOOLS\tools\timetest\timetest\scripts & python run_timetest.py timetest_infer_api_2.exe -d VPUX -niter 1 -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml"
Python 3.9.12
[setupvars.bat] OpenVINO environment initialized
[ INFO ] ========== cmd: C:\w_ov_dyn_win_t_230109_2200_2022_3_0_ci_tag_vpux_nightly_20230109_TOOLS\tools\timetest\timetest\scripts\timetest_infer_api_2.exe -m C:\models\mobilenet-v2\onnx\FP16\mobilenet-v2.xml -d VPUX -s C:\Users\LOCAL_~1\AppData\Local\Temp\tmp3ka806m1
[ INFO ] ========== Completed. Exit code: 0
[ DEBUG ] Statistics after run of executable #0: {'full_run': 7936950.0, 'first_inference_latency': 7445520.0, 'time_to_inference': 7373260.0, 'load_plugin': 1835060.0, 'create_exenetwork': 5517260.0, 'read_network': 174994, 'load_network': 5342260.0, 'first_inference': 72242, 'fill_inputs': 10923}
[ DEBUG ] Aggregated statistics after full run: {'full_run': {'avg': 7936950.0, 'stdev': 0}, 'first_inference_latency': {'avg': 7445520.0, 'stdev': 0}, 'time_to_inference': {'avg': 7373260.0, 'stdev': 0}, 'load_plugin': {'avg': 1835060.0, 'stdev': 0}, 'create_exenetwork': {'avg': 5517260.0, 'stdev': 0}, 'read_network': {'avg': 174994, 'stdev': 0}, 'load_network': {'avg': 5342260.0, 'stdev': 0}, 'first_inference': {'avg': 72242, 'stdev': 0}, 'fill_inputs': {'avg': 10923, 'stdev': 0}}
[ INFO ] Aggregated statistics:
{'create_exenetwork': {'avg': 5517260.0, 'stdev': 0},
'fill_inputs': {'avg': 10923, 'stdev': 0},
'first_inference': {'avg': 72242, 'stdev': 0},
'**first_inference_latency**': {'avg': 7445520.0, 'stdev': 0},
'full_run': {'avg': 7936950.0, 'stdev': 0},
'load_network': {'avg': 5342260.0, 'stdev': 0},
'load_plugin': {'avg': 1835060.0, 'stdev': 0},
'read_network': {'avg': 174994, 'stdev': 0},
'time_to_inference': {'avg': 7373260.0, 'stdev': 0}}

c:>

**Pull request merged**